deploy/crds/conversion_webhook_template.yaml

```diff
@@ -0,0 +1,22 @@
+# apiextensions.k8s.io/v1beta1
+  preserveUnknownFields: false
+  conversion:
+    strategy: Webhook
+    conversionReviewVersions: ["v1","v1alpha1","v1alpha2"]
+    webhookClientConfig:
+      service:
+        namespace: wildfly-op
```

Comment:
can we remove this namespace?
Depending on how the operator is installed it might end up in different namespaces (we'll have to figure out for OLM)